### PR TITLE
100.15.0 Release Branch

### PR DESCRIPTION
**Pull request by Gela**
# Description

Create a branch to merge 100.15 updates into main.

Assigning reviewers outside the SDK team since Tanner is currently on holiday.

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/EditData/ContingentValues/ContingentValues.pro [hunk 1]
```diff
@@ -28,6 +28,8 @@ TEMPLATE = app
 TARGET = ContingentValues
 
 ARCGIS_RUNTIME_VERSION = 100.14
+ARCGIS_RUNTIME_VERSION = 100.15
+ARCGIS_RUNTIME_VERSION = 100.15.0
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/EditData/ContingentValues/ContingentValues.pro:32]: we should just need one version defined (100.15.0)

File: ArcGISRuntimeSDKQt_CppSamples/Layers/ExportVectorTiles/ExportVectorTiles.pro [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+#-------------------------------------------------------------------------------
+
+CONFIG += c++14
+
+# additional modules are pulled in via arcgisruntime.pri
+QT += qml quick
+QT += opengl qml quick
+
+ARCGIS_RUNTIME_VERSION = 100.15
+include($$PWD/arcgisruntime.pri)
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Layers/ExportVectorTiles/ExportVectorTiles.pro:29]: this is duplicate of below

File: ArcGISRuntimeSDKQt_CppSamples/Maps/SetMaxExtent/setMaxExtent.pro [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+#-------------------------------------------------------------------------------
+
+CONFIG += c++14
+
+# additional modules are pulled in via arcgisruntime.pri
+QT += qml quick
+QT += opengl qml quick
+
+ARCGIS_RUNTIME_VERSION = 100.15.0
+include($$PWD/arcgisruntime.pri)
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Maps/SetMaxExtent/setMaxExtent.pro:29]: duplicate of below

File: sample-templates/templateArcGISRuntimeSDKQt_Cpp_Widgets/wizard.json [hunk 4]
```diff
@@ -0,0 +1,199 @@
+{
+    "version": 1,
+    "kind": "project",
+    "supportedProjectTypes": [ "Qt4ProjectManager.Qt4Project" ],
+    "id": "A.Sample_ArcGISQt_CPP__Widgets_100_14",
```
Comment (ldanzinger) [inline sample-templates/templateArcGISRuntimeSDKQt_Cpp_Widgets/wizard.json:5]: Seems like this should be updated to 100_15

File: sample-templates/templateArcGISRuntimeSDKQt_Cpp/wizard.json [hunk 5]
```diff
@@ -0,0 +1,231 @@
+{
+    "version": 1,
+    "kind": "project",
+    "supportedProjectTypes": [ "Qt4ProjectManager.Qt4Project" ],
+    "id": "A.Sample_ArcGISQt_CPP_100_14",
```
Comment (ldanzinger) [inline sample-templates/templateArcGISRuntimeSDKQt_Cpp/wizard.json:5]: Seems like this should be updated to 100_15

File: sample-templates/templateArcGISRuntimeSDKQt_QML/wizard.json [hunk 6]
```diff
@@ -0,0 +1,216 @@
+{
+    "version": 1,
+    "kind": "project",
+    "supportedProjectTypes": [ "Qt4ProjectManager.Qt4Project" ],
+    "id": "A.Sample_ArcGISQt_Qml_100_14",
```
Comment (JamesMBallard) [inline sample-templates/templateArcGISRuntimeSDKQt_QML/wizard.json:5]:
```suggestion
    "id": "A.Sample_ArcGISQt_Qml_100_15",
```

**Review by JamesMBallard — COMMENTED** [review]
The permissions changes should be reverted (`100755 → 100644`). Other than that, I have mostly the same comments as Lucas.

**Comment by Gela** [comment]
Confirmed with James that the permission change was intentional since the pri files don't need the executable permission and it is okay to keep. All other feedback has been addressed. Please review again @JamesMBallard @ldanzinger

**Pull request merged**